src/SumoCoders/FrameworkCoreBundle/Resources/assets/sass/_bootstrap-variables.scss

```diff
@@ -128,53 +137,53 @@ $caret-width-large:         5px !default;
 //## Customizes the `.table` component with basic values, each used across all table variations.
 
 //** Padding for `<th>`s and `<td>`s.
-$table-cell-padding:            8px !default;
+$table-cell-padding:            8px;
 //** Padding for cells in `.table-condensed`.
-$table-condensed-cell-padding:  5px !default;
+$table-condensed-cell-padding:  5px;
 
 //** Default background color used for all tables.
-$table-bg:                      transparent !default;
+$table-bg:                      transparent;
 //** Background color used for `.table-striped`.
-$table-bg-accent:               #f9f9f9 !default;
+$table-bg-accent:               #f3fcfd;
 //** Background color used for `.table-hover`.
-$table-bg-hover:                #f5f5f5 !default;
-$table-bg-active:               $table-bg-hover !default;
+$table-bg-hover:                #f3fcfd;
+$table-bg-active:               $table-bg-hover;
 
 //** Border color for table and cell borders.
-$table-border-color:            #ddd !default;
+$table-border-color:            #cedce2;
 
 
 //== Buttons
 //
 //## For each of Bootstrap's buttons, define text, background and border color.
 
-$btn-font-weight:                bold !default;
+$btn-font-weight:                600;
 
-$btn-default-color:              #333 !default;
-$btn-default-bg:                 #fff !default;
-$btn-default-border:             #ccc !default;
+$btn-default-color:              #7fa4b5;
+$btn-default-bg:                 $brand-default;
+$btn-default-border:             $btn-default-color;
 
-$btn-primary-color:              #fff !default;
-$btn-primary-bg:                 $brand-primary !default;
-$btn-primary-border:             darken($btn-primary-bg, 5%) !default;
+$btn-primary-color:              $white;
+$btn-primary-bg:                 #337ab7;
```

Comment:
de kleuren hier die terug komen mss ook bovenaan bij de kleuren zetten in een var